src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js

```diff
@@ -397,6 +401,16 @@ YmailApi.StormUI.prototype.displayFailure = function(message, opt_type) {
 };
 
 
+/**
+ * Log an interesting event
+ * @param {string} name
+ * @param {*} tags
```

Comment:
ok. will add a comment. do we have any more further restrictions on what can be put in tags? say, can we use array of string, i.e., calling it like `log('yme_compose', {recipients: ['a', 'b', 'c']})`